### ISSUE TITLE
ahoy init is not working

### ISSUE THREAD

**Issue by msound** [ahoy-cli/ahoy#32]
```
--- Projects/foo » ahoy
2016/07/02 12:31:59 AHOY! [fatal] ==>An ahoy config file couldn't be found in your path. You can create an example one by using 'ahoy init'
--- Projects/foo » ahoy init
2016/07/02 12:32:02 AHOY! [fatal] ==>An ahoy config file couldn't be found in your path. You can create an example one by using 'ahoy init'.
```

This was seen when using v1.1.0 (ahoy-darwin-amd64)

**Comment by frankcarey** [comment]
This should be fixed in the latest release `2.0.0-alpha2`.. you'll need to compile or install via homebrew `brew install --HEAD ahoy`

**Comment by mikeyjk** [comment]
FYI this page here still lists the version that results in this issue:
https://ahoy-cli.readthedocs.io/en/latest/#usage
